app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+from os import path
+
+
+def copy_file(command: str) -> None:
+    com, filename_in, filename_out = command.split()
```

Comment:
Use command instead of com, and you must check if command has exactly 3 arguments before it